EIPS/eip-2936.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 2936
+title: EXTCLEAR Opcode For SELFDESTRUCTed contracts
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-2936-extclear-for-selfdestruct/4569
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-03
+replaces: 2751
```

Comment:
```suggestion
```

This isn't a valid header field.

Remember, EIPs are just technical specifications, they don't compete against each other and there can be multiple EIPs that solve the same problem in different ways.  Most likely, only one (or potentially none!) will be implemented in Ethereum clients, but that is a separate process from the technical standards (EIP) process.

Reply:
`replaces` is in the [eip-template](https://github.com/ethereum/EIPs/blob/master/eip-template.md) and seems applicable here since 2751 would disable SELFDESTRUCT altogether.

Reply:
`replaces` is a valid field and should be used in conjunction with `superseded-by` (per EIP-1).

Reply:
OIC, I am using it inappropriately

Reply:
Actually I'm not sure anymore; does it require consent?

> EIPs may also have a `superseded-by` header indicating that an EIP has been rendered obsolete by a later document; the value is the number of the EIP that replaces the current do    cument. The newer EIP must have a `replaces` header containing the number of the EIP that it rendered obsolete.

Reply:
I think I'm interpreting this correctly because while it says `superceded-by` requires `replaces`, it does not say that `replaces` requires `superceded-by`.

Reply:
> replaces is a valid field and should be used in conjunction with superseded-by (per EIP-1).

Ugh, really?  Why haven't we got rid of that yet.  :cry:

----

Separately, you can only replace/supersede a final EIP, you definitely cannot replace an EIP that isn't even a draft yet (the bot won't even let anyone merge this).

Reply:
Okay, will rm